src/modules/addLike.js

```diff
@@ -0,0 +1,56 @@
+const id = 'Qtf3crftMlT7ljX0e5jU';
+const baseUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/';
+const url = `${baseUrl}apps/${id}/likes/`;
+
+const sendLike = async (like) => {
+  await fetch(url, {
+    method: 'POST',
+    body: JSON.stringify(like),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+};
+
+const retrieveLike = async (name) => {
+  const response = await fetch(url);
+  const data = await response.json();
+  const likedItem = data.find((item) => item.item_id === name);
+  const likesContainer = document.querySelector(`.${name}`);
+  likesContainer.innerHTML = likedItem.likes;
+};
+
+const addLike = async () => {
+  const likeButtons = document.querySelectorAll('.like');
+  likeButtons.forEach((btn) => {
+    btn.addEventListener('click', async (e) => {
+      const { name } = e.target.dataset;
+      const like = { item_id: name, name };
+      const { item_id: id } = like;
+      await sendLike(like);
+      retrieveLike(id, name);
+    });
+  });
+};
```

Comment:
optional

- [ ] I see your hard work here definitely you have invested time and mind there for user experience we will work on the future here I have one idea in my mind i.e  after hitting enter at 29 we could add style for `.${name}` class 
and get back as it was  after hitting enter on line 31  so for the required time we will have something to notify the user there like is counted